docs/feature_layers.md

```diff
@@ -53,7 +53,7 @@ There are a number of functions (and variables) related to how you can use or ma
 
 |Function                                      |Description                                                                                              |
 |----------------------------------------------|---------------------------------------------------------------------------------------------------------|
-| `layer_state_set(layer_mask)`                | Directly sets the layer state (recommended, do not use unless you know what you are doing).             |
+| `layer_state_set(layer_mask)`                | Directly sets the layer state (recommended not using it unless you know what you are doing).            |
```

Comment:
"recommended to avoid unless you know what you are doing" maybe?

Reply:
Yeah, that's better, i focused on not changing it much, but was still a bit confusing